src/components/icon/icons/Whatsapp.tsx

```diff
@@ -1,55 +1,10 @@
 export const Whatsapp = (props: React.SVGProps<SVGSVGElement>) => (
```

Comment:
Official name is `WhatsApp` with the A in capital letter

Reply:
Agree, but this icon is already being used in the platform. Do you recommend doing any kind of migration to that name?

Reply:
Oh true! then let keep it as it is